content/material_and_questions.md

```diff
@@ -13,23 +13,31 @@
 | 2026-2027 | John                                          | Narrative |
 | 2027-2028 | Hebrews, 1 Peter, 2 Peter                     | Epistle   |
 
-## Question Writing
+## Material
```

Comment:
Structurally, I think the "what is material" definition (maybe even under a "Material" header, should be above the Material Rotation Schedule, since one leads to the other

Reply:
Agreed.